src/pages/Events/Events.js

```diff
@@ -18,16 +19,16 @@ const Events = () => {
         <h4 className="events__subtitle">Upcoming Events</h4>
         <div className="events__cards">
           { events.map((event, i) =>
-            !event.acf.annual ? <CardEvent key={ i } props={ event } /> : ''
+            today < new Date(event.acf.event_date) ? <CardEvent key={ i } props={ event } /> : ''
           )}
         </div>
 
       </section>
       <section className="events">
-        <h4 className="events__subtitle events__subtitle--annual">Annual Events</h4>
+        <h4 className="events__subtitle events__subtitle--annual">Past Events</h4>
         <div className="events__cards">
-          { events.map((event, i) =>
-            event.acf.annual ? <CardEvent key={ i } props={ event } /> : ''
+          { events.map((event, i) => 
+            today > new Date(event.acf.event_date) ? <CardEvent key={ i } props={ event } /> : ''
```

Comment:
using an index as a key for maps is an [anti-pattern in react](https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9).